project/plugins.sbt

```diff
@@ -1 +1,2 @@
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.2")
+addSbtPlugin("com.eed3si9n" % "sbt-projectmatrix" % "0.7.0")
```

Comment:
Nice! didn't know that, I used to define subproject for each Scala version manually

Reply:
...although probably not needed now that you've used the `VirtualAxes` trick (another thing I didn't know, looks pretty useful)